README.md

```diff
@@ -135,20 +126,19 @@ $ mizu clean # mizu will continue running in cluster until clean is executed
 
 ## Configuration
 
-Mizu can work with config file which should be stored in ${HOME}/.mizu/config.yaml (macOS: ~/.mizu/config.yaml) <br />
-In case no config file found, defaults will be used <br />
+Mizu can optionally work with a config file that is either provided as a CLI argument or stored in ${HOME}/.mizu/config.yaml (macOS: ~/.mizu/config.yaml) or<br />
```

Comment:
Should we add the name of "CLI argument"

Reply:
added